From 3c87d4cf86de569c703a77617ed0894a559c6272 Mon Sep 17 00:00:00 2001
From: Carlos Soriano <csoriano@redhat.com>
Date: Fri, 30 Nov 2018 14:18:07 +0100
Subject: [PATCH 18/55] fileItem: Use similar function wording as signals for
 rename

---
 desktopManager.js | 2 +-
 fileItem.js       | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/desktopManager.js b/desktopManager.js
index 6814d1c..748f0f3 100644
--- a/desktopManager.js
+++ b/desktopManager.js
@@ -253,7 +253,7 @@ var DesktopManager = class {
             case RENAMED:
                 this._fileItems.delete(fileUri);
                 this._fileItems.set(otherFile.get_uri(), fileItem);
-                fileItem.renamed(otherFile);
+                fileItem.onFileRenamed(otherFile);
                 return;
             case CHANGES_DONE_HINT:
             case ATTRIBUTE_CHANGED:
diff --git a/fileItem.js b/fileItem.js
index 6b735e0..eeee2c1 100644
--- a/fileItem.js
+++ b/fileItem.js
@@ -201,6 +201,7 @@ var FileItem = class {
     }
 
     renamed(file) {
+    onFileRenamed(file) {
         if (this._queryFileInfoCancellable)
             this._queryFileInfoCancellable.cancel();
         this._queryFileInfoCancellable = new Gio.Cancellable();
-- 
2.20.0

