From f5344a8ddf0c8555c8dd457632bd2529c68a1593 Mon Sep 17 00:00:00 2001
From: Carlos Soriano <csoriano@redhat.com>
Date: Fri, 30 Nov 2018 15:51:22 +0100
Subject: [PATCH 21/55] fileItem: Reorder initialization, so UI is created
 before metadata

---
 fileItem.js | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/fileItem.js b/fileItem.js
index 4bbf563..1fd6fbc 100644
--- a/fileItem.js
+++ b/fileItem.js
@@ -76,8 +76,6 @@ var FileItem = class {
         else
             this._savedCoordinates = null;
 
-        this._updateMetadataFromFileInfo(fileInfo);
-
         this._state = State.NORMAL;
 
         this.actor = new St.Bin({ visible: true });
@@ -85,6 +83,7 @@ var FileItem = class {
         this.actor.set_height(Prefs.get_desired_height(scaleFactor));
         this.actor.set_width(Prefs.get_desired_width(scaleFactor));
         this.actor._delegate = this;
+        this.actor.connect('destroy', () => this._onDestroy());
 
         this._container = new St.BoxLayout({ reactive: true,
                                              track_hover: true,
@@ -103,15 +102,9 @@ var FileItem = class {
         this._container.add_actor(this._iconContainer);
 
         this._label = new St.Label({
-            text: this._displayName,
             style_class: 'name-label'
         });
 
-        if (this._isDesktopFile)
-            this._desktopFile = Gio.DesktopAppInfo.new_from_filename(this._file.get_path());
-
-        this._updateIcon();
-
         this._container.add_actor(this._label);
         let clutterText = this._label.get_clutter_text();
         /* TODO: Convert to gobject.set for 3.30 */
@@ -126,6 +119,14 @@ var FileItem = class {
 
         this._createMenu();
 
+        /* Set the metadata and update relevant UI */
+        this._updateMetadataFromFileInfo(fileInfo);
+
+        if (this._isDesktopFile)
+            this._desktopFile = Gio.DesktopAppInfo.new_from_filename(this._file.get_path());
+
+        this._updateIcon();
+
         this._isSelected = false;
         this._primaryButtonPressed = false;
         if (this._attributeCanExecute && !this._isDesktopFile)
@@ -153,7 +154,6 @@ var FileItem = class {
                 }
             });
         }
-        this.actor.connect('destroy', () => this._onDestroy());
     }
 
     _onDestroy() {
-- 
2.20.0

