From 72c23b0d6eabaa0673632a9937f12ced93d0dffd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20Dr=C4=85g?= <piotrdrag@gmail.com>
Date: Thu, 13 Dec 2018 18:28:00 +0100
Subject: [PATCH 51/55] desktopGrid: Fix spelling of OK in a user-visible
 string

---
 desktopGrid.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desktopGrid.js b/desktopGrid.js
index 4bdca0f..6d241f9 100644
--- a/desktopGrid.js
+++ b/desktopGrid.js
@@ -569,7 +569,7 @@ var RenamePopup = class {
                                            can_focus: true,
                                            x_expand: true });
         this._renameEntry.clutter_text.connect('activate', this._onRenameAccepted.bind(this));
-        this._renameOkButton= new St.Button({ label: _("Ok"),
+        this._renameOkButton= new St.Button({ label: _("OK"),
                                               style_class: 'app-view-control button',
                                               button_mask: St.ButtonMask.ONE | St.ButtonMask.THREE,
                                               reactive: true,
-- 
2.20.0

